src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -100,12 +103,18 @@ const SubsidiesSummary = ({
             className="border-0 shadow-none"
           />
         )}
-        {canEnrollWithEnterpriseOffers && (
+        {!hasAvailableLearnerCredit && canEnrollWithEnterpriseOffers && (
           <EnterpriseOffersSummaryCard
             className="border-0 shadow-none"
             offers={enterpriseOffers}
           />
         )}
+        { hasAvailableLearnerCredit && (
+          <LearnerCreditSummaryCard
+            className="border-0 shadow-none"
+            subsidyLearnerCredit={subsidyLearnerCredit}
+          />
+        )}
```

Comment:
[suggestion] Both the `EnterpriseOffersSummaryCard` and `LearnerCreditSummaryCard` are identical UI components, just backed by different data sources. Is there an opportunity to refactor/rename the existing `EnterpriseOffersSummaryCard` component to be used for both legacy enterprise offers and new Learner Credit, so we don't have duplicate UI components, just with different data sources?

For example, could the existing `EnterpriseOffersSummaryCard` be renamed to `LearnerCreditSummaryCard` and have the input data passed via props already transformed based on whether we're working with an enterprise offer or new Learner Credit? That way, we don't have 2 basically identical UI components (i.e., both say "Learner Credit" today).

Perhaps something similar to the following:

```jsx
function getLearnerCreditSummaryCardData({ enterpriseOffers, redeemableLearnerCreditPolicies} ) {
    const enterpriseOfferExpiringFirst = getOfferExpiringFirst(enterpriseOffers);
    const learnerCreditPolicyExpiringFirst = getPolicyExpiringFirst(redeemableLearnerCreditPolicies);
    return {
      expirationDate: learnerCreditPolicyExpiringFirst?.subsidyExpirationDate || enterpriseOfferExpiringFirst?.endDatetime,
    };
}

const SubsidiesSummary = () => {
  const {
    subscriptionPlan,
    subscriptionLicense: userSubscriptionLicense,
    couponCodes: { couponCodesCount },
    enterpriseOffers,
    canEnrollWithEnterpriseOffers,
    hasRedeemableLearnerCredit,
    redeemableLearnerCreditPolicies,
  } = useContext(UserSubsidyContext);

  const learnerCreditSummaryCardData = getLearnerCreditSummaryCardData({
    enterpriseOffers,
    redeemableLearnerCreditPolicies,
  });

  return (
    {(canEnrollWithEnterpriseOffers || hasAvailableLearnerCredit) && learnerCreditSummaryCardData.expirationDate && (
      <LearnerCreditSummaryCard
        className="border-0 shadow-none"
        expirationDate={learnerCreditSummaryCardData.expirationDate}
      />
    )}
  );
};
```

Reply:
> So, if an enterprise has both; offers and learner credit available, and the learner credit exists but it's end date value is null, it will show the expiration date of offers instead. Is that okay?

The majority, if not all, Learner Credit subsidies will have an expiration date. In the rare case a `Subsidy` does not have an expiration date and the enterprise also has a current offer with an expiration date, it would be sufficient to display the expiration date of the offer instead, so long as the offer is also current, as it would still be an applicable mechanism for the learner to redeem a course with, assuming the course is in the catalog associated with the offer.

Reply:
Makes sense.